app/config.py

```
@@ -1,468 +1,116 @@
 
-    Returns:
-        The string value of the variable, or default if unset/empty.
-    """
-    # os.environ.get() returns None for missing keys; the second os.getenv()
-    # reads whatever python-dotenv loaded from the .env file.
-    value = os.environ.get(name)
-    if value:
-        return value
-    return os.getenv(name, default) or default
+def _clean_redis_url(url: str) -> str:
+    if not url: return ""
```

Ruff (0.15.20)

[error] Multiple statements on one line (colon)

(E701)